src/arch/xtensa/xtos/xtos-internal.h

```diff
@@ -315,7 +315,7 @@ XTOS_PENDING_OFS:	.space	4	/* _xtos_pending variable */
 #  define IFNSA(a,b)	b
 # endif
 
-#define XTOS_TASK_CONTEXT_OFFSET	40
+#define XTOS_TASK_CONTEXT_OFFSET	48
```

Comment:
Is there a way to verify consistency of this number with C_ASSERT and offsetof in some place in .c code (or some asm version)?